src/GoTrueAdminApi.ts

```diff
@@ -157,7 +157,7 @@ export default class GoTrueAdminApi {
    * This function should only be called on a server. Never expose your `service_role` key in the browser.
    */
   async listUsers(): Promise<
-    { data: { users: User[] }; error: null } | { data: { users: [] }; error: AuthError }
+    { data: { users: User[], aud: string, next?: string | null, last?: string, total?: number }; error: null } | { data: { users: [] }; error: AuthError }
```

Comment:
You can use a type intersection here if you want to reuse the Pagination type defined below
```suggestion
    { data: { users: User[], aud: string } & Pagination; error: null } | { data: { users: [] }; error: AuthError }
```

Reply:
created an example in the ts playground here: https://www.typescriptlang.org/play?#code/C4TwDgpgBAYg9nKBeKBvAUFKAzAlgJwGdgA5AQwFsIAuKY-XAOwHN0BfddbAV0YGNguOIygAhMvgAUASlqooAEzLAycqN0IQitek2YBtALpQ2UAGSwEJtJij4IwbvhHylKtRq2FaRgDRY8IlJKGigAcgArOAALRjDrDjYgA